packages/ui-app/src/styles/app.css

```diff
@@ -73,7 +73,7 @@ article {
   border-radius: 0.25rem;
   box-sizing: border-box;
   margin: 0.25rem;
-  padding: 1.25rem 1.5rem;
+  padding: 1.25rem;
```

Comment:
Not convinced about this one, however does not seem to detract visually.

Reply:
Had to put a negative margin instead of a negative "top"/"right" to correclty displace the entire div, seems pretty consistent now, I would appreciate a second check tho :
![1](https://user-images.githubusercontent.com/40487685/62423250-1b079000-b6bf-11e9-93f0-5d56fefd7cbb.png)

Also added the mentioned ellipsis : 
![2](https://user-images.githubusercontent.com/40487685/62423251-1cd15380-b6bf-11e9-8fa2-e9ecd8812e50.png)

Reply:
Will take a peek thanks. (And thanks a million for the eliipsis as well, unexpected to see it in here, but make my account management much easier...)